docs/spec.md

```diff
@@ -2823,6 +2825,7 @@ More formally, `result[result_index] = operand[operand_index]` where:
 * `operand_index` = `add(full_start_index, full_offset_index)`.
   If `operand_index` is out of bounds for `operand`, then the behavior is
   implementation-defined.
```

Comment:
Also, what should happen to ScatterOp?

Reply:
HLO skips computing out of bounds indices: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/hlo/evaluator/hlo_evaluator.cc#L3020-L3025

Reply:
Alright, it looks like we may want to change the spec for ScatterOp as well, following the similar process as for GatherOp - first submit an interpreter PR as is, and then follow up with a delta.